cx_Freeze/freezer.py

```diff
@@ -191,6 +191,10 @@ def _CopyFile(self, source, target, copyDependentFiles,
         self.filesCopied[normalizedTarget] = None
         if copyDependentFiles \
                 and source not in self.finder.excludeDependentFiles:
+            # Always copy dependent files on root directory
+            # to allow to set relative reference
+            if sys.platform == 'darwin':
+                targetDir = self.targetDir
```

Comment:
Can you provide some additional information on why this is needed?

Reply:
Here: https://github.com/anthony-tuininga/cx_Freeze/blob/master/cx_Freeze/macdist.py#L195
The reference path is always set from @executable_path with only the file name.